bot/cogs/template.py

```diff
@@ -0,0 +1,14 @@
+from discord.ext import commands
+
+
+class Template:
```

Comment:
Cogs should have docstrings. If this is the Template, it's particularly important that it is well documented, so our other contributors will follow the example laid out by it.

Reply:
Now nicely documented showing examples of command groups and commands.